session.py

```diff
@@ -39,3 +39,24 @@ def request(self, verb, endpoint, params = {}, data = {}):
             raise Exception(f'{response.status_code}: {response.reason}')
 
         return response.json()
+
+    def RPCrequest(self, verb, endpoint, params = {}, data = {}):
```

Comment:
Rather than copy-pasting the entirety of the `request()` method, let's just add a single boolean `rpc` parameter, that toggles whether to use the RPC path or not.